doc/index.rst

```diff
@@ -2,17 +2,61 @@
 pyOptSparse
 ===========
 pyOptSparse is an object-oriented framework for formulating and solving nonlinear constrained optimization problems in an efficient, reusable, and portable manner.
-It is a fork of pyOpt that uses sparse matrices throughout the code to more efficiently handle large-scale optimization problems.
+Some key features of pyOptSparse include:
+
+- Object-oriented development maintains independence between the optimization problem formulation and its solution by different optimizers
+- Use of sparse matrices throughout the code to more efficiently handle large-scale optimization problems
+- Enable the use of optimizers when running in a MPI parallel environment, allows for evaluation of gradients in parallel, and can distribute function evaluations for gradient-free optimizers
+- Optimization solution histories can be stored during the optimization process. A partial history can also be used to warm-restart the optimization for any supported optimizer
+- A post-processing GUI utility called OptView to analyze optimization results
+
```

Comment:
Do you think it would sound better to rephrase the list items a bit like:
- Object-oriented development, which maintains [..]
- The use of sparse matrices [..] , to more efficiently [..]
- Enabling the use [...]
- The optimization solution histories [..]

It feels there is something slightly off in the current list, but I am not a native speaker so you do what you feel it's better

Reply:
Yeah I did not rewrite any of this, it was copy/pasted from the old document which was copy/pasted from the pyOpt README I think. Will reword some stuff.